pkg/rpc/server/server.go

```
@@ -1937,7 +1916,7 @@ drainloop:
 func (s *Server) blockHeightFromParam(param *request.Param) (int, *response.Error) {
 	num, err := param.GetInt()
 	if err != nil {
-		return 0, nil
+		return 0, response.ErrInvalidParams
```

Wow.

---

We need to return this error, is it OK?

---

Sure, it's a nice catch.